README.md

```diff
@@ -1,2 +1,4 @@
 # test-with-protectrepo
 test-with-protectrepo
+
+password: admin@123
```

Comment:
Suspeita de credencial encontrada, tipo: Password

Se for um falso positivo, adicione o seguinte na mensagem do commit (amend) para que seja ignorada:
```
FalsoPositivo: c205d48e
```